CruCentralCoast/Community/Ministry Teams/MinistryTeamsTVC.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  MinistryTeamsTVC.swift
+//  CruCentralCoast
+//
+//  Created by Landon Gerrits on 5/30/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+
+class MinistryTeamsTVC: UITableViewController {
```

Comment:
I'd remove a lot of the unnecessary comments and the places you have too many newlines